doc/source/index.rst

```diff
@@ -137,6 +134,8 @@ Getting Involved
 
    ray-overview/index.rst
    installation.rst
+   development.rst
```

Comment:
I don't think this should be at the top level, can we create a new section that is "Contributing" below "Other Libraries" that is...


.. toctree::
   development.rst
   getting-involved.rst

I realize we want to encourage contributions. However, we want to focus on **users** of Ray, not just those that contribute and therefore that initial page real estate is really important.